test/integration/visitor_not_authorized_to_view_user_data_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+
+class VisitorNotAuthorizedToViewUserDataTest < ActionDispatch::IntegrationTest
+  test "visitor cannot view other user's orders" do
+    user1, user2 = create_list(:user, 2)
```

Comment:
Useless assignment to variable - `user2`. Use `_` or `_user2` as a variable name to indicate that it won't be used.